recipes/torsiondrive/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "torsiondrive" %}
+{% set version = "0.8.2" %}
+{% set sha256 = "5fe4a238ffeaf35159bbe9556a37134475aeac0fd40bb69b8f9d538c46f2fc25" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: True  # [py<35]
```

Comment:
You can remove the `skip` and still use `noarch`. See:

https://conda-forge.org/docs/meta.html#building-noarch-packages

Reply:
This was updated. Thanks!